model/schema.sql

```diff
@@ -977,6 +977,72 @@ CREATE SQL SECURITY INVOKER VIEW `org_statistics`
    from organizations org;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `user_app`
+--
+
+DROP TABLE IF EXISTS `user_app`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `user_app` (
+  `uniqueId` varchar(255) CHARACTER SET utf8 NOT NULL,
```

Comment:
"CHARACTER SET utf8" means "COLLATE utf8_general_ci", that is, it chooses a case-insensitive collation. Probably not what we want here. Either "COLLATE utf8_bin", or, better, "COLLATE utf8mb4_bin".

("utf8" is a legacy encoding that only covers the Unicode BMP, utf8mb4 is the real UTF-8)